OZON.ru.js

```diff
@@ -9,596 +9,130 @@
 	"inRepository": true,
 	"translatorType": 4,
 	"browserSupport": "gcsibv",
-	"lastUpdated": "2014-04-03 18:47:00"
+	"lastUpdated": "2015-09-20 18:13:00"
 }
 
-/**
-	OZON.ru translator. Only supports "Book" "Item Type".
-	Copyright (c) 2012 Victor Rybynok, v.rybynok@gmail.com
 
-	Description
-	===========
```

Comment:
We'd like a license block. You can copy paste from https://github.com/zotero/translators/blob/master/ACM%20Digital%20Library.js#L15 and adjust authorship